e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -57,4 +58,26 @@ describe("scenarios > dashboard tabs", () => {
       cy.findByText("Orders").should("be.visible");
     });
   });
+
+  it("should only fetch cards on the current tab", () => {
```

Comment:
I think it would be nice to add request checks to make sure that card requests are only happening one time to catch regressions from future changes. This test current only checks that a card is requested on a new tab, not that it was not requested on a previous tab.

For example this could be nice to see:
|             | Navigate to Tab 1                                            | Navigate to Tab 2                                            |
|-------------|--------------------------------------------------------------|--------------------------------------------------------------|
| Tab 1 Cards | POST /api/dashboard/1/dashcard/1/card/1/query <br> (:heavy_check_mark: request sent) | POST /api/dashboard/1/dashcard/1/card/1/query <br> (:x:)    |
| Tab 2 Cards | POST /api/dashboard/1/dashcard/2/card/2/query <br>  (:x: not sent)     | POST /api/dashboard/1/dashcard/2/card/2/query <br>  (:heavy_check_mark:) |

With Cypress, are we able to check that certain requests have not occurred yet?

Reply:
Good idea, yeah we can check that requests haven't been sent. [Here](https://github.com/metabase/metabase/blob/f85ccd77fedf2264f0fe046e34527f024ffb62f0/e2e/test/scenarios/dashboard/dashboard.cy.spec.js#L447)'s an example in our codebase.